src/how-it-works.html

```diff
@@ -80,168 +118,118 @@ <h2>See BeakBox in action</h2>
 </section>
 
 <section>
-  <div class="layout-grid main-container">
-    <h2>No more cords!</h2>
-    <div class="flex">
-      <p>
-        With up to 5 days of battery life, BeakBox can now keep your parrots entertained with no cords attached. Enjoy the peace of mind that comes with knowing your birds are safe, stimulated, and
-        free from any risks.
-      </p>
-      <div class="flex">
-        <a class="button" href="https://shop.mybeakbox.com/" aria-label="">Shop BeakBox</a>
+  <div class="why-beakbox main-container main-container--narrow banner">
+    <h2>Why <span class="colored-text">choose BeakBox</span></h2>
+
+    <div class="flex-container-collapse">
+      <img src="/imgs/blurb-girl-green-parrot.png" alt="Girl holding a green parrot" />
+      <div class="content">
+        <div class="flex-container-collapse">
+          <div class="numbers">01</div>
```

Comment:
Does this need to be a list or at the very least moved to psuedo elements in CSS as it seems to be presentational and not important content?